cfgov/jinja2/v1/about-us/the-bureau/bureau-structure/_vars-division-data.html

```diff
@@ -20,9 +20,9 @@
         'name':   'Jeffrey Sumberg',
         'title':  'Human Capital'
         }, {
-        'titles': ['Chief Information Officer (CIO)', 'Assistant Director'],
+        'titles': ['Chief Information Officer (CIO)', 'Assistant Director<sup>*</sup>'],
         'slug':   'technology-and-innovation',
```

Comment:
Should we add "Acting" to the rest of the positions with the same status? We typically just rely upon the symbol / legend.